hoomd/hpmc/IntegratorHPMCMono.h

```diff
@@ -346,6 +346,16 @@ class IntegratorHPMCMono : public IntegratorHPMC
                     break;
                     }
                 }
+            m_requiresExpansiveSDF = false;
```

Comment:
There are no changes needed to this file or the `Shape*` files. As discussed above, detect based on the type of the `Shape` class.